common/meta/grpc/src/grpc_client.rs

```diff
@@ -346,7 +340,37 @@ impl MetaGrpcClient {
             let eps = self.endpoints.read().await;
             debug_assert!(!eps.is_empty());
 
-            self.conn_pool.get(&*eps).await?
+            let num_eps = eps.len();
+            let mut start = rand::thread_rng().gen_range(0..10);
```

Comment:
same, i think there is no need a random to loop the endpoints.

Reply:
Using random start to avoid pick up same endpoint everytime, and we just loop `endpoints.len()` times here to avoid repeating create connections all the time.